README.md

```diff
@@ -123,6 +126,9 @@ git submodule update --init --recursive
 set CMAKE_ARGS=-DONNX_USE_LITE_PROTO=ON
 pip install -e .
 ```
+
+Note: if you run into any issues while building ONNX, please ensure that the Python version for your activated conda environment is set to 3.8, and that python38.lib exists in the environment's folder on your device.
```

Comment:
Thank you for keeping improving README.md! but the build document should be more generic -- I would suggest:
- No specific Python version (except there is a specific issue for that version)
- Not assume users use Conda environment

May I understand the intention of this note? What shall we do if pythonxx.lib does not exist?

Reply:
Building ONNX and running the pip command returns the following error if there are conflicting versions of python. Since I was working with Anaconda 3.9, but my environment was in python 3.8, I ran into issues. If there are no assumptions that the user is working in a Conda environment or a specific Python environment, these changes are not necessary.

Should I close this PR, or are there any changes you recommend I make? Please let me know what you think is the best move.

Reply:
Thank you for providing the context. So it seems like a conflict issue about having multiple installed Python in certain environment. I might encounter similar issues before -- typically cleaning all prebuilt files (possibly there are some existing built files were generated by another Python environment) and rebuilding ONNX from source should help. Besides, if you use `pip install -e .` to build ONNX from source, you need to make sure your `pip` and `python` refer to the same installed Python. For instance, if you have one from native Python and another one from Conda Python with difference Python versions, you will probably see failures. If the issue you bumped into is reproducible, you can try whether the methods above help to resolve it.

> Should I close this PR, or are there any changes you recommend I make? Please let me know what you think is the best move.

Perhaps we can reword the sentences to make them more generic and then put them into [the section of common errors](https://github.com/onnx/onnx#common-errors). It can be something like: if you bump into issues like missing pythonXX.lib while install ONNX from source, please clean all existing build files and rebuild ONNX again. Also please make sure you have consistent Python version for common Python commands (e.g., `python` and `pip`).

Reply:
Yes, the "common errors" seems like a good place for these kind of troubleshooting hints (and FAQs).